doc/develop/extending.rst

```diff
@@ -391,13 +431,16 @@ To list all the properties of a component::
 .. note:
     When the name of a component which contains spaces is given as a 
     parameter, it should be put into quotes, such as ``"EBeam ExtXY"``.
+.. TODO last line too long
 
 To acquire
 5 images sequentially from the secondary electron detector at 5 different 
 positions on the sample, you could write this in bash::
 
     for i in $(seq 5); do odemis-cli --acquire "SED ExtXY" --output etd-pos$i.h5; odemis-cli --move OLStage y -100; done
 
+.. TODO line too long
```

Comment:
You could cut it the following way (and it should still copy-paste properly in a terminal):
```
for i in $(seq 5); do
   odemis-cli --acquire "SED ExtXY" --output etd-pos$i.h5
   odemis-cli --move OLStage y -100
done
```